src/Workspaces/Core/MSBuild/MSBuild/DiagnosticReportingOptions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.MSBuild
+{
+    internal struct DiagnosticReportingOptions
```

Comment:
>struct [](start = 13, length = 6)

Could be a readonly struct

Reply:
I suppose... if I knew how to use your new-fangled arcanery. :-P